src/operator/subgraph/subgraph_property.h

```diff
@@ -221,6 +221,10 @@ class SubgraphProperty {
     return nullptr;
   }
 
+  virtual void PrePartition() {}
```

Comment:
should we pass the graph to Pre/PostPartition functions?

we also need to pass the map of "options" to the subgraph prop somehow

Reply:
added graph and options_map as parameters